src/test/test-argv-util.c

```diff
@@ -115,6 +115,15 @@ TEST(rename_process) {
         test_rename_process_one("1234567", 1); /* should always fit */
 }
 
+TEST(argv_help) {
+        assert_se(argv_looks_like_help(1, STRV_MAKE("program", NULL))); /* No argument */
+        assert_se(argv_looks_like_help(2, STRV_MAKE("program", "help", NULL))); /* First argument is "help" */
+        assert_se(argv_looks_like_help(3, STRV_MAKE("program", "arg1", "--help", NULL))); /* Second argument is "--help" */
+        assert_se(argv_looks_like_help(4, STRV_MAKE("program", "arg1", "arg2", "-h", NULL))); /* Third argument is "-h" */
+        assert_se(!argv_looks_like_help(2, STRV_MAKE("program", "arg1", NULL))); /* Argument list doesn't contain "help", "--help" or "-h" */
+        assert_se(!argv_looks_like_help(4, STRV_MAKE("program", "arg1", "arg2", "--h", NULL))); /* Argument list doesn't contain "help", "--help" or "-h" */
```

Comment:
The trailing `NULL`s in `STRV_MAKE()` are not necessary. Please drop them.
I.e. for example, `STRV_MAKE("program")`, instead of `STRV_MAKE("program", NULL)`.

Reply:
On it!

This community is awesome!